.github/workflows/nightly.yml

```diff
@@ -36,3 +36,13 @@ jobs:
         run: |
           bin/flake8
           bin/coverage run bin/test -vv1
+
+      - name: Slack Notification
+        uses: rtCamp/action-slack-notify@v2
+        env:
+          SLACK_CHANNEL: alert-engineering
+          SLACK_COLOR: ${{ job.status }} # or a specific color like 'green' or '#ff00ff'
+          SLACK_MESSAGE: 'Job status was:  ${{ job.status }}'
+          SLACK_TITLE: Build outcome for crate-python
+          SLACK_USERNAME: jenkins
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
```

Comment:
I am really wondering if this will work. I used those credentials for testing purposes here as they are currently already used for the very same purpose, on a different CI system. However, I don't know if `SLACK_WEBHOOK` actually means `SLACK_TOKEN`?

If notifications will actually work, we will create a dedicated account for github vs. jenkins before propagating this to more repositories, right? Maybe we can even add both `SLACK_USERNAME` and `SLACK_WEBHOOK` as organization-wide secrets in order to ease maintenance.